tests/unit/shared/dbgoutput.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include <fstream>
+
+#include "../../src/Debug/debugger.h"
+
+class DBGOutput {
```

Comment:
Why is there code for debug tests @carllocos? This is what we developed [latch](https://github.com/TOPLLab/latch) for?